warehouse/packaging/models.py

```diff
@@ -399,7 +400,21 @@ def latest_version(self):
             .first()
         )
 
-
+    def is_verified_url(self, url: str) -> bool:
+        return (
+                orm.object_session(self)
+                .query(File.Event)
+                .join(File)
+                .join(Release)
+                .join(Project)
+                .filter(Project.id == self.id)
+                .filter(
+                    literal(url).ilike(File.Event.additional.op("->>")("publisher_url")+"%")
+                    )
```

Comment:
Might want to run `make reformat` on these files, I think this should be:
```suggestion
                )
```